internal/cmd/base/server_test.go

```diff
@@ -900,3 +907,24 @@ func TestSetupWorkerPublicAddress(t *testing.T) {
 		})
 	}
 }
+
+func TestSetupWorkerCredentialStorage(t *testing.T) {
+	ctx := context.Background()
+	assert, require := assert.New(t), require.New(t)
+	ts := db.TestWrapper(t)
+	keyId, err := ts.KeyId(ctx)
+
+	tw := worker.NewTestWorker(t, &worker.TestWorkerOpts{
```

Comment:
I'm not sure this is a valid test... you're testing a `TestWorker` which is completely different than testing a `Worker`.  Perhaps we need to test both?

Reply:
Doesn't a `TestWorker` create a new Worker with some default settings using the `New()` function from `worker.go`?

Reply:
Ah, ty!  I didn't read the func far enough down to see that call.